tools/shared.py

```diff
@@ -2536,6 +2536,13 @@ def make_shared_library(js_file, wasm_file):
     f.close()
     return wso
 
+def asbytes(s):
+  if str is bytes:
```

Comment:
Shouldn't it be `if s is bytes` ?

Reply:
`str is bytes` checks whether the current Python version uses byte string by default.